docs/api/source/index.rst

```diff
@@ -1,13 +1,102 @@
 :orphan:
 
-==============================
-solid-client-access-grants API
-==============================
+====================================================
+@inrupt/solid-client-access-grants API Documentation
+====================================================
 
+``@inrupt/solid-client-access-grants`` is a JavaScript library for
+requesting and managing access given to an agent for a resource. These
+access grants are signed by a Solid Pod owner’s private key, which
+itself is managed by the Solid Pod server.
+
+It is part of a `family open source JavaScript
+libraries <https://docs.inrupt.com/developer-tools/javascript/client-libraries/>`__
+designed to support developers building Solid applications.
+
+.. admonition:: Server Support
+   :class: caution
+
+   This experimental feature is currently only available in `Inrupt Enterprise
+   Solid Server (ESS)
+   <https://www.inrupt.com/products/enterprise-solid-server>`__.
+
+Installation
+------------
+
+For the latest stable version of solid-client-access-grants:
+
+.. code:: bash
+
+   npm install @inrupt/solid-client-access-grants
+
+Changelog
+~~~~~~~~~
+
+See `the release
+notes <https://github.com/inrupt/solid-client-access-grants-js/blob/main/CHANGELOG.md>`__.
+
+
+Browser Support
+~~~~~~~~~~~~~~~
+
+Our JavaScript Client Libraries use relatively modern JavaScript
+features that will work in all commonly-used browsers, except Internet
+Explorer. If you need support for Internet Explorer, it is recommended
+to pass them through a tool like `Babel <https://babeljs.io>`__, and to
+add polyfills for e.g. ``Map``, ``Set``, ``Promise``, ``Headers``,
+``Array.prototype.includes``, ``Object.entries`` and
+``String.prototype.endsWith``.
+
+Additionally, when using this package in an environment other than Node.js, you
+will need `a polyfill for Node's buffer module
+<https://www.npmjs.com/package/buffer>`__.
```

Comment:
Is this the case for this library too, or is it only for solid-client-authn ?

Reply:
I'm not certain, tbh, so I'm kinda just going with "standardise this across projects" for now